src/main/java/org/cascadebot/cascadebot/commands/fun/ColourCommand.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2019 CascadeBot. All rights reserved.
+ * Licensed under the MIT license.
+ */
+
+
+package org.cascadebot.cascadebot.commands.fun;
+
+import net.dv8tion.jda.core.entities.Member;
+import org.cascadebot.cascadebot.commandmeta.CommandContext;
+import org.cascadebot.cascadebot.commandmeta.ICommandMain;
+import org.cascadebot.cascadebot.commandmeta.Module;
+import org.cascadebot.cascadebot.permissions.CascadePermission;
+
+import java.awt.*;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class ColourCommand implements ICommandMain {
+
```

Comment:
Add a check to see if there are no arguments provided, if so replyUsage. Otherwise, it errors with the infamous index out of bounds.